Examples/Multiplayer/CSharp/Foundation-Unity-Example/Assets/Scripts/HelloWorld.cs

```diff
@@ -248,9 +248,17 @@ private async Task LoginAsync(string email, string password)
 
         using CspSystems.LoginStateResult loginResult = await userSystem.Login(string.Empty, email, password, true);
 
-        // Cache user ID for later use.
```

Comment:
If its okay with the CSP team, I would prefer we don't include changes to the examples yet until we have verified the refactor works as expected fully in the OKO App. My intent is to have this update as a separate ticket once the update is complete internally.

Reply:
Sounds good, we will keep that in mind for the next pass of changes, thank you